lib/cylc/scheduler.py

```diff
@@ -293,7 +293,8 @@ def _start_print_blurb():
         )
         cylc_license = """
 The Cylc Suite Engine [%s]
-Copyright (C) 2008-2018 NIWA & British Crown (Met Office) & Contributors.
+Copyright (C) 2008-2018 NIWA & British Crown
+(Met Office) & Contributors.
```

Comment:
Can we break at the `&` operator?

```
Copyright (C) 2008-2018 NIWA
& British Crown (Met Office) & Contributors.
```

Reply:
[W503](http://pycodestyle.pycqa.org/en/latest/intro.html?highlight=W503#error-codes) or [W504](http://pycodestyle.pycqa.org/en/latest/intro.html?highlight=W504#error-codes)?